demo/priv/repo/seeds.exs

```diff
@@ -2,7 +2,7 @@ import Demo.Factory
 
 category_names = ["Tech", "Misc", "Crew", "News"]
 
-categories =
+_categories =
   Enum.reduce(category_names, [], fn name, acc ->
     [insert(:category, name: name) | acc]
   end)
```

Comment:
I guess, you removed this because you wanted to use Ash for the `category` resource?

Reply:
I reversed this change.